symfony/plugins/orangehrmPerformancePlugin/entity/PerformanceTracker.php

```diff
@@ -0,0 +1,239 @@
+     */
+    private Employee $addedBy;
+
+    /**
+     * @var Collection
+     *
+     * @ORM\OneToMany (targetEntity="OrangeHRM\Entity\PerformanceTrackerReviewer", mappedBy="performanceTracker")
+     */
+    private $PerformanceTrackerReviewer;
+
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+        //$this->PerformanceTrackerLog = new ArrayCollection();
+        $this->PerformanceTrackerReviewer = new ArrayCollection();
+    }
+
+    /**
+     * @return int
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * @param int $id
+     */
+    public function setId(int $id): void
+    {
+        $this->id = $id;
+    }
+
+    /**
+     * @return string
+     */
+    public function getTrackerName(): string
+    {
+        return $this->trackerName;
+    }
+
+    /**
+     * @param string $trackerName
+     */
+    public function setTrackerName(string $trackerName): void
+    {
+        $this->trackerName = $trackerName;
+    }
+
+    /**
+     * @return DateTime
+     */
+    public function getAddedDate(): DateTime
+    {
+        return $this->addedDate;
+    }
+
+    /**
+     * @param DateTime $addedDate
+     */
+    public function setAddedDate(DateTime $addedDate): void
+    {
+        $this->addedDate = $addedDate;
+    }
+
+    /**
+     * @return DateTime|null
+     */
+    public function getModifiedDate(): ?DateTime
+    {
+        return $this->modifiedDate;
+    }
+
+    /**
+     * @param DateTime|null $modifiedDate
+     */
+    public function setModifiedDate(?DateTime $modifiedDate): void
+    {
+        $this->modifiedDate = $modifiedDate;
+    }
+
+    /**
+     * @return int|null
+     */
+    public function getStatus(): ?int
+    {
+        return $this->status;
+    }
+
+    /**
+     * @param int|null $status
+     */
+    public function setStatus(?int $status): void
+    {
+        $this->status = $status;
+    }
+
+    /**
+     * @return Employee
+     */
+    public function getEmployee(): Employee
+    {
+        return $this->employee;
+    }
+
+    /**
+     * @param Employee $employee
+     */
+    public function setEmployee(Employee $employee): void
+    {
+        $this->employee = $employee;
+    }
+
+    /**
+     * @return Collection
+     */
+    public function getPerformanceTrackerReviewer()
+    {
+        return $this->PerformanceTrackerReviewer;
+    }
+
+    /**
+     * @param Collection $PerformanceTrackerReviewer
+     */
+    public function setPerformanceTrackerReviewer($PerformanceTrackerReviewer): void
+    {
+        $this->PerformanceTrackerReviewer = $PerformanceTrackerReviewer;
+    }
+
+    /**
+     * @return Employee
+     */
+    public function getAddedBy(): Employee
+    {
+        return $this->addedBy;
+    }
+
+    /**
+     * @param Employee $addedBy
+     */
+    public function setAddedBy(Employee $addedBy): void
+    {
+        $this->addedBy = $addedBy;
+    }
+
+
+
```

Comment:
Remove blank lines and add new line to end of file

Reply:
done